README.md

```diff
@@ -1,9 +1,10 @@
 # doobie
 
 <img align="right" src="https://cdn.rawgit.com/tpolecat/doobie/series/0.3.x/doobie_logo.svg" height="150px" style="padding-left: 20px"/>
-[![Travis CI](https://travis-ci.org/tpolecat/doobie.svg?branch=master)](https://travis-ci.org/tpolecat/doobie)
+Release:[![Travis CI](https://travis-ci.org/tpolecat/doobie.svg?branch=master)](https://travis-ci.org/tpolecat/doobie)
+0.4.x:[![Travis CI](https://travis-ci.org/tpolecat/doobie.svg?branch=0.4.0-staging)](https://travis-ci.org/tpolecat/doobie)
```

Comment:
to clarify, you mean change from `0.4.0-staging` to 0.4.x` (implying that this branch will eventually be renamed/promoted to `0.4.x`) and to get rid of the master branch?

Only concern with getting rid of the master badge would be that when this branch gets merged into master, if no one remembers to add the master badge back in then we will remove the master status badge from the readme.